Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -406,6 +424,41 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
         updateNavigationTitle()
     }
 
+    open func channelController(
+        _ channelController: _ChatChannelController<ExtraData>,
+        didChangeTypingMembers typingMembers: Set<_ChatChannelMember<ExtraData.User>>
+    ) {
+        let typingMembersWithoutCurrentUser = typingMembers.filter { $0.id != self.channelController.client.currentUserId }
+        
+        if typingMembers.isEmpty {
```

Comment:
Shouldn't this be `typingMembersWithoutCurrentUser`?  
If only the current user is typing, it will be in `typingMembers`, so this condition will hit the `else` branch but you will pass empty set into `showTypingIndicator`, which will use typingUnknown and show "Someone is typing" :D

Reply:
yep I fixed this already, but thanks :D